website/content/docs/entrypoint/disable.mdx

```diff
@@ -55,3 +55,28 @@ app "my-app" {
 
 For builders that do not support automatic injection, you must manually
 alter your build scripts to prevent entrypoint installation.
+
+## Disabling Specific Features
+
+### Exec
+
+You can disable only the [`waypoint exec`](/docs/exec) functionality by
+setting the `WAYPOINT_CEB_DISABLE_EXEC` environment variable to a truthy
+value (e.g. "1"). Other entrypoint features such as logs, application config,
+etc. will continue to work.
```

Comment:
Might be worth calling out explicitly where this needs to be set. It looks like it needs to be done in some plugin `env` block in waypoint.hcl, and `waypoint config set WAYPOINT_CEB_DISABLE_EXEC=true` doesn't because it runs before the CEB can pull down env vars from the server? I can also imagine someone setting this at build time or on the server though and expecting it to work.

Reply:
I do already in the last paragraph. Should I move that up?

Reply:
I moved it up and added an example for k8s